docs/gitbook/tutorials/osm-progressive-delivery.md

```diff
@@ -331,10 +339,10 @@ Exec into the load tester pod with:
 kubectl -n test exec -it flagger-loadtester-xx-xx sh
 ```
 
-Repeatedly generate 404s:
+Repeatedly generate 404s until canary rollout fails:
 
 ```bash
-watch -n 1 curl http://podinfo-canary.test:9898/status/404
+watch -n 0.1 curl http://podinfo-canary.test:9898/status/404

```

Comment:
@stefanprodan on testing this, I found that sometimes the failure was quite close to the thresholds and took a long while to generate canary rollback. @johnsonshi suggested changing this to 0.1, which helped mitigate both those problems, hope this is okay.
For example:
![image](https://user-images.githubusercontent.com/42152676/143088832-1f5bf355-80be-4ae9-afde-ac8c3372e388.png)

Reply:
Yeah numbers higher than 0.1 would result in the failure experiment not being demonstrated. As we can see here, with the command being repeated `0.1` times per second, it was still pretty close!